projects/services/public-api-service-typescript/build.sh

```diff
@@ -7,4 +7,7 @@ yarn install
 
 yarn run build
 
+echo "Copy mock server from client-sdk-javascript to run contract test..."
+cp ../../../client/client-sdk-javascript/test/mock-server.ts test
```

Comment:
It would probably make more sense (but not a showstopper) to move mock-server to a `common` folder, or (maybe even better) move to the server-side as a `testkit` and have it exported, hence available, to the client.

Reply:
I agree, it should be extracted later. I think @electricmonk can take it from here because he's working on the contract test a the moment.